packages/client/src/pages/Home/components/Rating/Rating.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { FC } from 'react'
+import { RatingProps } from './types'
+import { clsx } from 'clsx'
+import style from './rating.module.css'
+import Divider from '../../../../components/Divider/Divider'
+import { tableRows } from './constants'
+
+const Rating: FC<RatingProps> = ({ className }) => {
+  return (
+    <div className={clsx(style.containerRating, className)}>
+      <Divider className={style.divider} />
+      <div className={style.title}>рейтинг в игре</div>
+      <div className={style.text}>
+        Попади в <span>TOP</span>
```

Comment:
Здесь ссылка на рейтинг

```suggestion
        Попади в <NavLink to={Routes.LeaderBoard}>TOP</NavLink >
```